### ISSUE TITLE
Which headers do I need to expose for session cookies?

### ISSUE THREAD

**Issue by alexweissman** [palanik/CorsSlim#13]
I'm having trouble persisting sessions across requests made via cross-origin requests.  I'm starting to suspect this has something to do with the way the response headers are modified by CorsSlim.

Do I need to set `exposeHeaders`?  If so, which headers do I need to expose for sessions to work properly?  Just the `Set-Cookie` header?

**Comment by palanik** [comment]
Have you tried [Access-Control-Allow-Credentials](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Allow-Credentials)?

**Comment by alexweissman** [comment]
Yep, I eventually figured out that I needed to set `Access-Control-Allow-Credentials` using the `withCredentials` parameter in CorsSlim, as well as the `withCredentials` XHR parameter in my AJAX request.  See http://stackoverflow.com/a/24689738/2970321

However, it appears that Firefox blocks third-party cookies by default, which is why I was having trouble even after I set the necessary headers.  So, I cannot rely on third-party session cookies for mission-critical functionality.  I'll have to find a more RESTful way to implement my API.  Thanks!
